src/general/MainMenu.tscn

```diff
@@ -903,6 +903,7 @@ margin_top = 28.0
 margin_bottom = 628.0
 
 [node name="GalleryViewer" parent="." instance=ExtResource( 17 )]
+Exclusive = false
```

Comment:
I think we have more non-exclusive popups so having the exclusive property default to false would be good in my opinion.